src/radical/pilot/configs/resource_aliases.json

```diff
@@ -1,33 +1,14 @@
 
 {
     "aliases": {
-        "das4.fs2"                          : "das4.fs2_ssh",
-        "das5.fs1"                          : "das5.fs1_ssh",
-        "epsrc.archer"                      : "epsrc.archer_orte",
         "fub.allegro"                       : "fub.allegro_rsh",
```

Comment:
Is this current?

Reply:
fs1 and 2 don't exist anymore, archer has changed completely by now, any configuration we have is outdated.